bigquery-antipattern-recognition/src/main/java/com/google/zetasql/toolkit/antipattern/parser/Main.java

```diff
@@ -94,9 +112,15 @@ public static void main(String[] args) throws ParseException, IOException {
     logger.info(
         "Processing finished."
             + "Queries read: {}. "
+            + "Fully parsed queries: {}. "
+            + "Partially parsed queries: {}. "
+            + "Queries that could not be parsed: {}. "
             + "Queries with anti-patterns: {}. "
-            + "Queries that could not be parsed: {}.",
+            + "Queries with another error: {}.",
```

Comment:
What is the difference between "that could not be parsed" and "another error"

Reply:
Could not be parsed is if 0 parsers were able to parse the query.
Other error is if no issue is found with the parsers, but the analysis falls in the global catch block. Then we can know that some queries had problems.